lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -170,95 +177,122 @@ def install_libraries
               pod_target.umbrella_header_path
             end.compact.group_by(&:dirname)
 
-            pod_targets.sort_by(&:name).each do |pod_target|
+            pod_target_installation_results = Hash[pod_targets.sort_by(&:name).map do |pod_target|
               target_installer = PodTargetInstaller.new(sandbox, pod_target, umbrella_headers_by_dir)
-              target_installer.install!
+              [pod_target.name, target_installer.install!]
```

Comment:
any reason to key by the name instead of the target itself?

Reply:
uh no really I had initially fallen into the trap it wont be hashable...